lib/restforce/middleware/raise_error.rb

```diff
@@ -16,11 +16,18 @@ def on_complete(env)
     end
 
     def message
-      "#{body.first['errorCode']}: #{body.first['message']}"
+      "#{body['errorCode']}: #{body['message']}"
     end
 
     def body
-      JSON.parse(@env[:body])
+      @body = (@env[:body].respond_to?(:first) ? @env[:body].first : @env[:body])
```

Comment:
This didn't entirely fix my issue as ActiveRecord actually monkey patch String to add the `first` method. :cry:

I'll add a PR to fix this again.